control/README.md

```diff
@@ -1,6 +1,42 @@
-# Filament Ethereum Outpost
+# Filament Ethereum Control
 
 ## Token grants
 
 The implementation of token grants is a modified version of the
 [starknet token](https://github.com/starknet-io/StarkNet-Token).
+
+## Developement
+
+Install foundry:
+
+```Bash
+curl -L https://foundry.paradigm.xyz | bash
+```
+
+## Deploy
+
+### Local
+
+Deploying contracts locally makes use of `anvil`.
+
+After starting anvil it will print some accounts and private keys which can be
+used to author transactions. With the default configuration the server will
+listen on `localhost:8545`.
+
+#### Token
+
+The following will deploy the token contract and mint tokens to the account
+specified via the `TOKEN_RECEIVER` variable.
+
+```Bash
+PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 TOKEN_RECEIVER=0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 TOKEN_MINT_AMOUNT=10000000000000000000000 forge script script/FilamentToken.s.sol:DeployFilaTokenScript --rpc-url http://localhost:8545 --broadcast
```

Comment:
Missing a `cd` into `control` subdir, otherwise the command will fail.

Reply:
this readme file is in the control directory, so I assume that it was clear but will add.